bosh/jobs.yml

```diff
@@ -0,0 +1,79 @@
+meta:
+  docker_templates:
+  - name: docker
+    release: docker
+  - name: newrelic-monitor
+    release: newrelic
+  - name: collectd
+    release: collectd
+  hardening_templates: (( merge ))
+
+jobs:
+- name: docker-compute
+  templates: (( meta.docker_templates meta.hardening_templates ))
+  instances: (( merge ))
```

Comment:
@jmcarp These `instances: <X>` properties are set to a number from the secrets. With a number of instances set to `instances: 0`, it shouldn't need any changes to `deployments.yml`.